indra/tests/test_reach.py

```diff
@@ -53,11 +63,52 @@ def test_parse_site_residue_only():
     text = ['serine residue', 'serine', 'a serine site', 's', 'ser']
     assert unicode_strs(text)
     for t in text:
-        residue, site = ReachProcessor._parse_site_text(t)
+        sites = ReachProcessor._parse_site_text(t)
+        assert(len(sites) == 1)
+        residue, site = sites[0]
         assert unicode_strs((residue, site))
         assert(residue == 'S')
         assert(site is None)
 
+def test_parse_site_multiple():
+    sites = ReachProcessor._parse_site_text('638/641')
+    assert(len(sites) == 2)
+    assert(sites[0][0] == None)
+    assert(sites[0][1] == '638')
+    assert(sites[1][0] == None)
+    assert(sites[1][1] == '641')
+
+    sites = ReachProcessor._parse_site_text('992,1068')
+    assert(len(sites) == 2)
+    assert(sites[0][0] == None)
+    assert(sites[0][1] == '992')
+    assert(sites[1][0] == None)
+    assert(sites[1][1] == '1068')
+
+    sites = ReachProcessor._parse_site_text('Y1221/1222')
+    assert(len(sites) == 2)
+    assert(sites[0][0] == 'Y')
+    assert(sites[0][1] == '1221')
+    assert(sites[1][0] == None)
+    assert(sites[1][1] == '1222')
+
+    sites = ReachProcessor._parse_site_text('Tyr-577/576')
+    assert(len(sites) == 2)
+    assert(sites[0][0] == 'Y')
+    assert(sites[0][1] == '577')
+    assert(sites[1][0] == None)
```

Comment:
Could you actually make this 'Y' instead of None? The pattern implies that both of those residues are tyrosine (though possibly on different isoforms of the protein agent).